recipe/activate.bat

```diff
@@ -0,0 +1,9 @@
+if defined CONDA_BUILD_STATE (
```

Comment:
I'm not sure when `CONDA_BUILD_STATE` is set, but it seems this is shown every time when I launch an Anaconda command prompt. Maybe this fix is needed?:


```suggestion
@echo off
if defined CONDA_BUILD_STATE (
```

Reply:
Thanks! Perhaps Windows has `echo` defaulted to `on`.

Reply:
`CONDA_BUILD_STATE` is set in a `conda build` environment, i.e. when building the recipe, see 
https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html